docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+Then you can close the browser tab. Just return to the terminal and you will see ``Success! Credentials have been saved.``
+
+After successful authentication access token will be saved in your config. This token will be used by ``legionctl`` for all
+API calls to the EDI server under the hood:
+
+.. code-block:: console
+
+    $ legionctl config get edi_access_token --show-secrets
+
+`Using jupyterlab`
+
+The method of authentication depends on how to your Jupyter Lab is configured. Read more about different auth methods
+with Jupyter Lab plugin.
+
+Prepare code and data
+######################
+
+Prepare MLFlow project
+------------------------
+
+In this tutorial we will solve the famous `hello world` problem of wine classification using Legion Platform with MLFlow integration.
+
+First of all let's review project source code.
+To to this you should clone legion-examples github repository. This repo includes examples of projects that are ready to be used
+with Legion Platform. We will focus on wine example that is located at ``mlflow/sklearn/wine/`` directory inside the repo.
+So let's also change our directory to that after repo cloning and look around with ``tree`` command
+
+.. code-block:: console
+
+   $ git clone https://github.com/legion-platform/legion-examples && cd legion-examples
+   $ cd mlflow/sklearn/wine/
+   $ tree
+
+We will se next structure of project
+
+.. code-block:: text
+    :caption: ../mlflow
+    :name: MLproject example file
+
+    .
+    ├── MLproject
+    ├── airflow
+    │   └── dag.py
+    ├── conda.yaml
+    ├── data
+    │   └── wine-quality.csv
+    ├── legion
+    │   ├── deployment.legion.yaml
+    │   ├── packaging.legion.yaml
+    │   ├── training.legion.yaml
+    │   └── wine_connection.legion.yaml
+    ├── test
+    │   └── test_wine.py
+    ├── train.py
+    └── wine-quality.csv
+
+In this directory, actually only some files are interested for us in the context of the tutorial:
+
+- Three files that belongs to MLFlow project itself (see MLFlow docs for extra info)
+    - MLproject
+    - conda.yaml
+    - train.py
+- Legion yaml files that we will use as a payload for Legion API in training, packaging and deployment stages
+    - legion/training.legion.yaml
+    - legion/packaging.legion.yaml
+    - legion/deployment.legion.yaml
+    – legion/wine_connection.legion.yaml
+    – legion/vcs_connection.legion.yaml
+    – legion/docker_connection.legion.yaml
+– File where input data for learning is saved
+    – data/wine-quality.csv
+
+To learn more about MLFlow project structure you can use next official MLFlow documentation.
+
+We'll show this files with a bit comments.
+
+.. code-block:: console
+
+   $ cat MLproject
+
+.. code-block:: yaml
+    :caption: MLproject
+    :name: MLproject example file
+
+    name: wine-quality-example
+    conda_env: conda.yaml
+    entry_points:
+        main:
+            parameters:
```

Comment:
Could you please describe all fields in general (w/ appropriate MLFlow docs link) and describe how they are connected w/ Legion?

Reply:
We should describe the connection in the legion API files.